gui/src/2D/controls/control.ts

```diff
@@ -1853,6 +1853,16 @@ export class Control {
         if (canNotify && this.parent != null) { this.parent._onWheelScroll(deltaX, deltaY); }
     }
 
+    /** @hidden */
+    public _onCanvasBlur(): void {
```

Comment:
What happens if I click a button, keep the mouse button clicked, move outside of the window and get back to the window?

Reply:
https://www.babylonjs-playground.com/#HH8Q22

Try with the color picker for instance. Currently I can pick a color, maintain the mouse button and go outside of the window

Reply:
Fair point, with this solution we would have to implement _onCanvasBlur for every control which uses pointerUp to handle going outside the window as well.

The easy solution would be to just trigger onPointerUp for those controls, but the problem with that is _onPointerUp expects coordinates, which a canvasBlur event wouldn't have.

Not sure what the best approach is...I can definitely implement _onCanvasBlur for all the controls which need it if we think that's the right solution.

Reply:
I'm more on the fence to do nothing for all controls and start by having it  do something only for slider for now

Reply:
I would think maybe we should fix it for the color picker as well since it is possible to get the color picker "stuck" by releasing the mouse outside the window, just like the slider. I don't think we need to fix it for the button neccessarily.

Reply:
Did you manage to get it stuck?

Reply:
The color picker currently gets stuck if I click on it, switch to a new tab, release the mouse button, and then come back. We could fix this by just calling _forcePointerUp I think.

Reply:
You are right! Let's do it :)

Reply:
Done! Let me know what you think